Add new selectData, fix styles selectorCategories

Abramov21 opened the pull request.

Abramov21 committed:
Add new selectData, fix styles selectorCategories

001elijah committed:
Merge branch 'main' into CashFlowSection

001elijah reviewed (changes requested):
є конфлікти, не можу змерджити. розписав рішення у Slack

Abramov21 committed:
add logic, fix style

001elijah committed:
update Cashflow

update FinanceDataBoard.jsx, FormCashflow.jsx, SelectCategory.jsx, TextDataInput.jsx, authSlice.js

001elijah reviewed (dismissed):
в тебе були незакриті помилки, я дещо повиправляв, дещо закоментував. ці моменти треба дивитися, бо якщо помилки на гілці є і далі це змерджити, то мейн не змонтується.

001elijah committed:
Merge branch 'main' into CashFlowSection
